src/main/scala/com/raphtory/core/client/RaphtoryGraph.scala

```diff
@@ -80,6 +82,12 @@ private[core] class RaphtoryGraph[T: ClassTag: TypeTag](
   logger.info(s"Created Graph object with deployment ID '$deploymentID'.")
   logger.info(s"Created Graph Spout topic with name '$spoutTopic'.")
 
+  try {
+    new HTTPServer(8899)
```

Comment:
can we please make the port be pulled from the config? Is there also any reason why we are not storing the HTTP server object? what if we want to shut it down?

Reply:
Sounds great, I've stored and stopped it post execution as well